Python_Engine/Query/Version.cs

```diff
@@ -48,12 +48,19 @@ public static PythonVersion Version(string pythonExecutable)
                 }
             };
             string versionString;
-            using (Process p = Process.Start(process.StartInfo))
+            try
```

Comment:
This method should check if the file exists before even making the `Process` object.